queue/src/fiskaltrust.Middleware.Localization.v2/JournalProcessor.cs

```diff
@@ -136,50 +129,48 @@ private List<T> GetConfigurationFromDictionary<T>(string key)
 
     }
 
-    private async IAsyncEnumerable<JournalResponse> ToJournalResponseAsync<T>(IAsyncEnumerable<T> asyncEnumerable, int chunkSize)
+    private PipeReader ToJournalResponse<T>(IAsyncEnumerable<T> asyncEnumerable)
     {
-        using var memoryStream = new MemoryStream();
-        using var writer = new StreamWriter(memoryStream);
-        using var jsonWriter = new JsonTextWriter(writer);
-        var serializer = new JsonSerializer();
-        serializer.Serialize(jsonWriter, await asyncEnumerable.ToArrayAsync().ConfigureAwait(false));
-        jsonWriter.Flush();
-        if (memoryStream.Length < chunkSize)
-        {
-            yield return new JournalResponse
-            {
-                Chunk = memoryStream.ToArray().ToList()
-            };
-        }
-        else
+        Pipe response = new();
+
+        Task.Run(async () =>
         {
-            memoryStream.Seek(0, SeekOrigin.Begin);
-            var buffer = new byte[chunkSize];
-            int readAmount;
-            while ((readAmount = await memoryStream.ReadAsync(buffer, 0, buffer.Length).ConfigureAwait(false)) > 0)
+            try
             {
-                yield return new JournalResponse
+                await foreach (var journal in asyncEnumerable)
                 {
-                    Chunk = buffer.Take(readAmount).ToList()
-                };
-                buffer = new byte[chunkSize];
+                    try
+                    {
+                        await response.Writer.WriteAsync(Encoding.UTF8.GetBytes(JsonConvert.SerializeObject(journal)));
```

Comment:
JsonConvert or JsonSerailizer?

Reply:
I did not change that behavior. If there was a reason to use newtonsoft here it's still valid. (but IDK if there was a reason ^^)